_episodes/09-under-the-hood.md

```diff
@@ -66,7 +71,24 @@ All branches and tags in Git are pointers to commits.
 
 ---
 
-## Demonstration: experimenting with branches
+### Git is basically a content-addressed storage system
+
+- CAS: ["mechanism for storing information that can be retrieved based on its content, not its storage location"](https://en.wikipedia.org/wiki/Content-addressable_storage)
+- content address is the content digest (SHA-1 checksum)
+- stored data does not change - so when we modify commits, we always create new commits
+
+Let us poke a bit into raw objects! Start with:
+
+
+```shell
+$ git cat-file -p HEAD
+```
+
+Then explore the `tree` object, then the `file` object, etc.
```

Comment:
... by using each of the hashes that come up when you do the exploration.

I will make my own PR now for both.